src/routes/console/project-[project]/overview/platforms/wizard/web/step2.svelte

```diff
@@ -12,8 +12,12 @@
 
     let method: Method = Method.NPM;
 
-    const example1 = `import { Appwrite } from 'appwrite';`;
-    const example2 = `<script src="https://cdn.jsdelivr.net/npm/appwrite@${$versions['client-web']}" />`;
+    const example1 = `import { Client } from 'appwrite';`;
+    const example2 =
+        `<script src="https://cdn.jsdelivr.net/npm/appwrite@${$versions['client-web']}" />
+<script>
+    const { Client } = Appwrite;
+</script` + `>`; // Prevent svelte from processing the closing script tag
```

Comment:
btw if I try to do:

```
</script>`;
```

this happens:

<img width="801" alt="image" src="https://user-images.githubusercontent.com/1477010/203163585-5ea41453-f336-48d1-8a96-8bdea32098f7.png">